integration-tests/tests/run-notebook.js

```diff
@@ -9,50 +9,82 @@ const testRunNotebookFn = _.flow(
   withWorkspace,
   withBilling,
   withRegisteredUser
-)(async ({ workspaceName, page, testUrl, token }) => {
+)(async ({ billingProject, workspaceName, page, testUrl, token }) => {
+  await testRunNotebookHelper(billingProject, workspaceName, page, testUrl, token)
+})
+
+const testRunNotebookWithV1WorkspaceFn = _.flow(
+  withV1Workspace,
+  withBilling,
+  withRegisteredUser
+)(async ({ billingProject, v1WorkspaceName, page, testUrl, token }) => {
+  await testRunNotebookHelper(billingProject, v1WorkspaceName, page, testUrl, token, false)
+})
+
+const testRunNotebookHelper = async (billingProject, workspaceName, page, testUrl, token, newEntity = true) => {
```

Comment:
I'd like to see this defined above its usages. Also, I'm not sure that `newEntity` is the right name for this, maybe like `isWorkspaceSingleUse`?

Reply:
Done